docs/setup_installation/aws/getting_started.md

```diff
@@ -25,104 +30,11 @@ to security reasons.
   </iframe>
 </p>
 
-To create a cross-account role for [managed.hopsworks.ai](https://managed.hopsworks.ai), you need our AWS account id and the external
-id we created for you. You can find this information on the first screen of the cross-account
-configuration flow. Take note of the account id and external id and go to the *Roles* section
-of the *IAM* service in the AWS Management Console and select *Create role*.
-
-<p align="center">
-  <figure>
-    <a  href="../../../assets/images/setup_installation/managed/aws/create-role-instructions.png">
-      <img style="border: 1px solid #000;width:700px" src="../../../assets/images/setup_installation/managed/aws/create-role-instructions.png" alt="Creating the cross-account role instructions">
-    </a>
-    <figcaption>Creating the cross-account role instructions</figcaption>
-  </figure>
-</p>
-
-Select *Another AWS account* as trusted entity and fill in our AWS account id and the external
-id generated for you:
-
-<p align="center">
-  <figure>
-    <a  href="../../../assets/images/setup_installation/managed/aws/create-role-aws-step-1.png">
-      <img style="border: 1px solid #000;width:700px" src="../../../assets/images/setup_installation/managed/aws/create-role-aws-step-1.png" alt="Creating the cross-account role step 1">
-    </a>
-    <figcaption>Creating the cross-account role step 1</figcaption>
-  </figure>
-</p>
-
-Go to the last step of the wizard, name the role and create it:
-
-<p align="center">
-  <figure>
-    <a  href="../../../assets/images/setup_installation/managed/aws/create-role-aws-step-2.png">
-      <img style="border: 1px solid #000;width:700px" src="../../../assets/images/setup_installation/managed/aws/create-role-aws-step-2.png" alt="Creating the cross-account role step 1">
-    </a>
-    <figcaption>Creating the cross-account role step 2</figcaption>
-  </figure>
-</p>
-
-As a next step, you need to create an access policy to give [managed.hopsworks.ai](https://managed.hopsworks.ai) permissions to manage
-clusters in your organization's AWS account. By default, [managed.hopsworks.ai](https://managed.hopsworks.ai) is automating all steps required to launch
-a new Hopsworks cluster. If you want to limit the required AWS permissions, see [restrictive-permissions](restrictive_permissions.md).
-
-Copy the permission JSON from the instructions:
+In [managed.hopsworks.ai](https://managed.hopsworks.ai/) click on *Connect to AWS* or go to *Settings* and click on *Configure* next to *AWS*. Then click on *Cross-accont role*. This will direct you to a page with the instructions needed to create the Cross account role and set up the connection. Follow the instructions.
```

Comment:
I think we should also add screenshots of the cross-account role page on managed.hopsworks.ai here as well

Reply:
Do you mean the setting page or the one where we set the values for the cross account? Because if it is the second the screenshot is present (line 41)

Reply:
yes, can you update it to be on the first step instead on the Step 3

Reply:
to clarify, what i meant is that this screentshot on line 41 that needs updating to be focused on the first step and also since the text about access key is removed
![save-role](https://raw.githubusercontent.com/logicalclocks/logicalclocks.github.io/main/docs/assets/images/setup_installation/managed/aws/save-role.png)